src/Administration/Resources/app/administration/src/module/sw-settings-language/snippet/en.json

```
@@ -13,14 +13,18 @@
       "columnIsoCode": "ISO Code ",
       "columnInherit": "Inherit from",
       "columnDefault": "Default",
+      "defaultLabel": "Default",
       "buttonAddLanguage": "Add language",
       "contextMenuEdit": "Edit",
       "messageDeleteSuccess": "Language \"{name}\" has been deleted.",
       "buttonDelete": "Delete",
       "titleSidebarFilter": "Filter",
       "titleSidebarQuickFilter": "Quick filter",
       "textFilterRootLanguages": "Show only \"root\" languages",
-      "textFilterInheritedLanguages": "Show only inherited languages"
+      "textFilterInheritedLanguages": "Show only inherited languages",
+      "manageSnippets": "Manage snippets",
+      "cardTitle": "Languages",
+      "cardSubtitle": "Languages can be assigned to Sales Channel domains in order to display the appropriate translations. They also enable the translation of content in categories, products, shopping experience pages, etc."
```

```suggestion
      "cardSubtitle": "Languages can be assigned to Sales Channel domains in order to display the appropriate translations. They also enable the translation of content in categories, products, Shopping Experience pages, etc."
```

Shopping Experiences is a marketing term, therefore the "declared" way to write it (even capital cases) is always the correct one